src/components/organisms/Badges/BadgeImage.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useCallback, useRef } from "react";
+
+const defaultAvatar = "/icons/sparkle-nav-logo.png";
+
+export const BadgeImage: React.FC<{
+  image?: string;
+}> = ({ image }) => {
+  const imageRef = useRef<HTMLImageElement>(null);
+
+  const changeBadgeImage = useCallback(() => {
+    if (!imageRef.current) return;
+
+    imageRef.current.src = defaultAvatar;
+  }, []);
```

Comment:
Do you need this? Can you just pass `image ?? defaultAvatar` directly into the `src=` prop in `img`?

Reply:
`onError` is fired if the image URL doesn't exist. We will need both your suggestion and the ref solution, to ensure the `img` tag always has a valid avatar.